pkg/providers/instancetype/suite_test.go

```diff
@@ -466,6 +466,7 @@ var _ = Describe("InstanceType Provider", func() {
 			nodeClass.Spec.Kubelet = &v1alpha2.KubeletConfiguration{
 				ImageGCHighThresholdPercent: lo.ToPtr(int32(30)),
 				ImageGCLowThresholdPercent:  lo.ToPtr(int32(20)),
+				FailSwapOn:                  lo.ToPtr(false), // opposite of default
```

Comment:
Should we add comments for this in case of other boolean defaults to take it obvious that we are testing something other than the default?

Reply:
Yes, opportunistically (I had this fixed for CPUCFSQuota)